projects/samples/contests/robocup/protos/RobocupGoal.proto

```diff
@@ -0,0 +1,268 @@
+}%
+
+Solid {
+  translation IS translation
+  rotation IS rotation
+  children [
+    DEF HORIZONTAL_POST_TRANS Transform {
+      translation 0 0 %{=postHeight}%
+      rotation 0 0 1 1.5708
+      children [
+        Shape {
+          appearance DEF POST_APP PBRAppearance {
+            baseColor IS frameColor
+            metalness 0
+            roughness 0.2
+          }
+          geometry Capsule {
+            height %{=postSpacing}%
+            radius %{=postRadius}%
+          }
+        }
+      ]
+    }
+    DEF RIGHT_POST_TRANS Transform {
+      translation %{=-postSpacing/2}% 0  %{=postHeight/2}%
+      rotation 1 0 0 1.5708
+      children [
+        Shape {
+          appearance USE POST_APP
+          geometry Cylinder {
+            bottom FALSE
+            top FALSE
+            height %{=postHeight}%
+            radius %{=postRadius}%
+          }
+        }
+      ]
+    }
+    DEF LEFT_POST_TRANS Transform {
+      translation %{=postSpacing/2}% 0  %{=postHeight/2}%
+      rotation 1 0 0 1.5708
+      children [
+        Shape {
+          appearance USE POST_APP
+          geometry Cylinder {
+            bottom FALSE
+            top FALSE
+            height %{=postHeight}%
+            radius %{=postRadius}%
+          }
+        }
+      ]
+    }
+    Shape {
+      appearance DEF NET_APP PBRAppearance {
+        baseColor IS netColor
+        baseColorMap ImageTexture {
+          url "../../../../objects/robotstadium/protos/textures/net.png"
+          filtering 0
+        }
+        metalness 0
+        roughness 1
+      }
+      geometry IndexedFaceSet {
+        coord Coordinate {
+          point [
+            %{=-postSpacing/2}% %{=-C}%          0
+            %{=postSpacing/2}%  %{=-C}%          0
+            %{=postSpacing/2}%  %{=-C}%          %{=postHeight}%
+            %{=-postSpacing/2}% %{=-C}%          %{=postHeight}%
+            %{=postSpacing/2}%  %{=-postRadius}% 0
+            %{=-postSpacing/2}% %{=-postRadius}% 0
+            %{=postSpacing/2}%  %{=-postRadius}% %{=postHeight}%
+            %{=-postSpacing/2}% %{=-postRadius}% %{=postHeight}%
+          ]
+        }
+        texCoord TextureCoordinate {
+          point [
+            0 0
+            28 0
+            28 16
+            0 16
+            7 0
+            7 16
+            0 16
+            28 0
+            28 6
+            0 6
+          ]
+        }
+        coordIndex [
+          0, 1, 2, 3, -1,
+          3, 2, 1, 0, -1,
+          1, 4, 6, 2, -1,
+          2, 6, 4, 1, -1,
+          0, 5, 7, 3, -1,
+          3, 7, 5, 0, -1,
+          2, 3, 7, 6, -1,
+          6, 7, 3, 2, -1
+        ]
+        texCoordIndex [
+          0, 1, 2, 3, -1,
+          0, 1, 2, 3, -1,
+          0, 4, 5, 6, -1,
+          0, 4, 5, 6, -1,
+          0, 4, 5, 6, -1,
+          0, 4, 5, 6, -1,
+          0, 7, 8, 9, -1,
+          0, 7, 8, 9, -1
+        ]
+      }
+    }
+    Transform {
+      translation 0 %{=-C}%  %{=postHeight}%
```

Comment:
Here there is an additional space between the values.
Additionally as, in a previous comment we should add more spaces in the lua code
```suggestion
      translation 0 %{=-C}% %{=postHeight}%
```